### PR TITLE
Agregar definicion y ejemplo.

### PR DESCRIPTION
**Pull request by dbarrerap**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by dbarrerap** [comment]
I guess Flask wont make it in Spanish...

File: guide/spanish/flask/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+necesario__ para poner en funcionamiento una aplicaci&oacute;n 
+b&aacute;sica en cuesti&oacute;n de minutos.
+
+Incluye un servidor web de desarrollo para probar la aplicaci&oacute;n 
+sin necesidad de instalar algo como Apache o Nginx. Tambien trae un 
+depurador y soporte integrado para pruebas unitarias.
+
+Otro de los elementos mas &uacute;tiles de Flask es el __esquema de 
+rutas__ y documentaci&oacute;n. Con el decorador de rutas la 
+aplicaci&oaacute;n puede responder a peticiones totalmente RESTful.
```
Comment (lynxlynxlynx) [inline guide/spanish/flask/index.md]: the acute isn't rendered due to a typo:
```suggestion
aplicaci&oacute;n puede responder a peticiones totalmente RESTful.
```

**Comment by lynxlynxlynx** [comment]
Other than that, LGTM! Very nice job.

**Comment by RandellDawson** [comment]
@lynxlynxlynx Are these changes you can make on behalf of the user or should we close due to lack of response?

**Comment by lynxlynxlynx** [comment]
One syntax error is not the end of the world. Thanks for the PR, dbarrerap.

**Pull request merged**